test/recipes/03-test_fipsinstall.t

```diff
@@ -30,6 +30,9 @@ my $infile = bldtop_file('providers', platform->dso('fips'));
 my $fipskey = $ENV{FIPSKEY} // config('FIPSKEY') // '00';
 my $provconf = srctop_file("test", "fips-and-base.cnf");
 
+run(test(["fips_version_test", "-config", $provconf, "<3.4.0"]),
```

Comment:
I am doing this version check above the skip block because their is an env variable set that causes the fips_version_test to fail otherwise.